### PR TITLE
support webauth; update README with webauthn code sample

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -639,6 +639,138 @@ if (idxResponse.isLoginSuccessful()) {
 ```
 [//]: # (end: loginUsingPasswordAndPhoneAuthenticator)
 
+#### Login using password + web authenticator
```
Comment (vijetmahabaleshwar-okta) [inline README.md:642]: We can link this document on how to extract webauth data from browser - https://developer.okta.com/docs/reference/api/authn/#get-the-signed-assertion-from-the-webauthn-authenticator

Comment (arvindkrishnakumar-okta) [reply README.md:642]: done

File: README.md [hunk 2]
```diff
@@ -639,6 +639,140 @@ if (idxResponse.isLoginSuccessful()) {
...
+        .filter(x -> "select-authenticator-authenticate".equals(x.getName()))
+        .findFirst();
+remediationOption = remediationOptionsOptional.get();
+
+// get authenticator options
+authenticatorOptions = remediationOption.getAuthenticatorOptions();
+log.info("Authenticator Options: {}", authenticatorOptions);
+
+// select webauthn (fingerprint) authenticator
+Authenticator fingerPrintAuthenticator = new Authenticator();
```
Comment (vijetmahabaleshwar-okta) [inline README.md:726]: Technically webauthn can be any of the biometric/security authenticators like yubikey, face scan etc. It's better to rename `fingerPrintAuthenticator` as `webauthnAuthenticator` instead. 
That brings back the question whether the fields `authenticatorData` etc are specific to fingerprint only. But that's a different question

**Review by vijetmahabaleshwar-okta — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply README.md:726]: good question, let me see if I can find an answer.

**Pull request merged**